functions.php

```diff
@@ -49,24 +49,67 @@ function dt_pcsu_signup_blog_notification_email_rename( $message, $domain, $path
-  <label for="dt_prayer_site">
-    Do you have an existing prayer website? What is the link?
-  </label>
-  <input type="text" id="dt_prayer_site" name="dt_prayer_site">
-  <label for="dt_reason_for_subsite">
-    What is this prayer campaign for?
-  </label>
-  <input type="text" id="dt_reason_for_subsite" name="dt_reason_for_subsite">
-  <p>
-    <label for="dt_newsletter">
-      <input id="dt_newsletter" type="checkbox" name="dt_newsletter" checked>
-      <strong>Sign me up for Pray4Movement news.</strong>
+    <style>
+        #privacy { display: none}
+        .private-notice { color: #949494 }
+    </style>
+    <br>
+    <br>
+    <br>
+    <label for="dt_champion_name">
+        What is your name? <span class="private-notice">Answer is kept private.</span>
     </label>
-  </p>
+    <input type="text" id="dt_champion_name" name="dt_champion_name">
+    <label for="dt_prayer_site">
+        Do you have an existing prayer network? If so, what is the link? <span class="private-notice">Answer is kept private.</span>
+    </label>
+    <input type="text" id="dt_prayer_site" name="dt_prayer_site">
+    <label for="dt_reason_for_subsite">
+        What is your target location or people group? <span class="private-notice">Answer is kept private.</span>
+    </label>
+    <input type="text" id="dt_reason_for_subsite" name="dt_reason_for_subsite">
+    <p>
+        <label>Chose a Campaign Type:</label>
+            <?php
+            // @todo enable this
+            // $wizard_types = apply_filters( 'dt_campaigns_wizard_types', [] );
+            $wizard_types = [
+                'ongoing-porch' => [
+                    'campaign_type' => 'ongoing',
+                    'porch' => 'ongoing-porch',
+                    'label' => '24/7 Ongoing Campaign',
+                ],
+                '24hour' => [
+                    'campaign_type' => '24hour',
+                    'porch' => 'generic-porch',
+                    'label' => '24/7 Campaign with a start and end date'
+                ],
+                'ramadan-porch' => [
+                    'campaign_type' => '24hour',
+                    'porch' => 'ramadan-porch',
+                    'label' => '24/7 Ramadan Template',
+                ],
+            ];
+            foreach ( $wizard_types as $type => $type_value ): ?>
+                <label>
+                    <input type="radio" name="porch_type" value="<?php echo esc_html( $type ); ?>" required>
+                    <?php echo esc_html( $type_value['label'] ); ?>
+                </label>
+            <?php endforeach; ?>
+
+    </p>
+
+
+    <p>
+        <label for="dt_newsletter">
+            <input id="dt_newsletter" type="checkbox" name="dt_newsletter" checked>
+            <strong>Sign me up for Pray4Movement news.</strong>
+        </label>
+        <label for="p4m_agreement">
+            <input id="p4m_agreement" type="checkbox" name="p4m_agreement" checked required>
```

Comment:
should it start unchecked, like most other websites do, so that they have to actively read and click on the agreement?

Reply:
Ah, good point.
We don't have anything else right now for them to read but that line.